commit d03971dac7b32a6ffcbf161853e017f65ae7c22f
Author: Heiko Becker <heirecka@exherbo.org>
Date:   Fri Feb 11 21:35:54 2022 +0100

    meson: Don't build lame plugin with -Dlame=disabled
    
    Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/1686>

diff --git a/ext/lame/meson.build b/ext/lame/meson.build
index 2169fde6f4..3290f17e1e 100644
--- a/ext/lame/meson.build
+++ b/ext/lame/meson.build
@@ -1,5 +1,10 @@
+lame_dep = dependency('', required: false)
 lame_option = get_option('lame')
 
+if lame_option.disabled()
+  subdir_done()
+endif
+
 lame_extra_c_args = []
 lame_dep = cc.find_library('mp3lame', required: false)
 have_lame = cc.has_header_symbol('lame/lame.h', 'lame_init')
